### ISSUE TITLE
Load private keys into `--suri` from a file

### ISSUE THREAD

**Issue by ltfschoen** [paritytech/cargo-contract#1106]
In one of the YouTube videos by Andrew Jones, he mentioned that instead of uploading a contract with `cargo contract upload --suri //Alice` for example, it was preferable to use the contracts-ui to avoid exposing private key (if it wasn't one of the build-in accounts).

Are we able to populate the private keys into a text file and interpret its contents via the ``suri` option of the cargo contract command similar to like i did [here](https://github.com/ltfschoen/Edgeware-seed-generating-script-SSSS/blob/master/edgeware_wallets.sh#L23)

**Comment by ascjones** [comment]
Yes please Luke. I don't have time to do this in the near future but would love to see it if anyone wants to pick it up!

rel. https://github.com/paritytech/cargo-contract/issues/674

**Comment by ltfschoen** [comment]
> Yes please Luke. I don't have time to do this in the near future but would love to see it if anyone wants to pick it up!
> 
> rel. #674

I've tried to do it in this PR https://github.com/paritytech/cargo-contract/pull/1134 and it seems to be working fine

**Comment by ltfschoen** [comment]
@ascjones FYI, i don't think this is a necessary feature since you can just load the secrets from and environment variable using a bash script or similar like this https://github.com/ltfschoen/validator-scripts?tab=readme-ov-file#rotate-validator-session-keys-with-via-cli-with-polkadotjs-tools-instead-of-via-ui-with-polkadotjs-apps
